### PR TITLE
Fetch backup opensea key

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: rotkehlchen/externalapis/opensea.py [hunk 1]
```diff
@@ -85,6 +88,44 @@ def __init__(self, database: 'DBHandler', msg_aggregator: MessagesAggregator) ->
...
+            return self.backup_key
+
+        try:
+            response = requests.get('https://rotki.com/api/1/credentials', json={'name': 'opensea'})  # noqa: E501
+        except requests.exceptions.RequestException as e:
+            log.error(f'Could not connect to rotki server for fetching backup opensea key due to {str(e)}')  # noqa: E501
+            return None
+
+        try:
+            json_ret = json.loads(response.text)
```
Comment (yabirgb) [inline rotkehlchen/externalapis/opensea.py:105]: Just as a note here you can do that raises json.decoder.JSONDecodeError
```suggestion
            json_ret = response.json()
```

File: rotkehlchen/externalapis/opensea.py [hunk 2]
```diff
@@ -85,6 +88,44 @@ def __init__(self, database: 'DBHandler', msg_aggregator: MessagesAggregator) ->
...
+        except JSONDecodeError as e:
+            log.error(f'Could not decode rotki server response for opensea key: {response.text} due to {str(e)}')  # noqa: E501
+            return None
+
+        if response.status_code != 200:
```
Comment (yabirgb) [inline rotkehlchen/externalapis/opensea.py:110]: Shouldn't this happen before trying to decode the json response? If something goes wrong you will see the error related to the json response but not that it was caused by a non 200

File: rotkehlchen/externalapis/opensea.py (hunk 1 above)
Comment (LefterisJP) [reply rotkehlchen/externalapis/opensea.py:105]: I know. I also prefer response.json(), this was a copy paste.

File: rotkehlchen/externalapis/opensea.py (hunk 2 above)
Comment (LefterisJP) [reply rotkehlchen/externalapis/opensea.py:110]: Not necessarily as our server should always return json. Returns 4XX for errors and contains an "error" key in the dict.

**Review by LefterisJP — COMMENTED** [review]
Gonna merge as all is green and it works.

**Pull request merged**